src/data/ACTIONS/root/DNC.ts

```diff
@@ -38,6 +38,7 @@ export const DNC = ensureActions({
 		gcdRecast: 2500,
 		speedAttribute: Attribute.SKILL_SPEED,
 		cooldown: 30000,
+		cooldownGroup: 15997, // Finishing Move shares a cooldown with Standard Step
```

Comment:
Is the CDG here arbitrary? I assume it doesn't matter as long as there isn't a conflict with another CGD, but the xiv-id lookup page says the CDG for Standard Step/Finishing Move is 7.

Reply:
I feex